fuzzers/libfuzzer_libmozjpeg/src/lib.rs

```diff
@@ -128,7 +129,8 @@ fn fuzz(corpus_dirs: &[PathBuf], objective_dir: PathBuf, broker_port: u16) -> Re
     let mut fuzzer = StdFuzzer::new(scheduler, feedback, objective);
 
     // The wrapped harness function, calling out to the LLVM-style harness
-    let mut harness = |buf: &[u8]| {
+    let mut harness = |input: &BytesInput| {
+        let buf = input.bytes();
```

Comment:
`target_bytes`?

Reply:
@andreafioraldi does the split beween `bytes` and `target_bytes` makes sense atm? One was for the target, one for serialization?

Reply:
Maybe we can remove `HasTargetBytes` and only use `HasBytesVec`, not sure though

Reply:
No HasBytesVec is for Inputs that have a vec field and so returns a reference to these bytes. HasTargetBytes is for serialization to the bytes consumed by the target, and can be an owned type for the Inputs that produce the vector only when serializing (e.g. a grammar based Input)